app-emulation/box64/autobuild/defines

```diff
@@ -0,0 +1,33 @@
+PKGNAME=box64
+PKGSEC=utils
+PKGDES="an userspace x86_64 emulator with a twist, aimed to run x86_64 Linux programs on non-x86_64 Linux systems"
+PKGDEP="glibc gcc-runtime"
+ABTYPE="cmakeninja"
+BUILDDEP="mold"
+
+FAIL_ARCH="(amd64|loongson3)"
```

Comment:
This is a false assumption - we have many more architectures than just the six we have included for the mainline so far. If the fact is that "only four architectures are supported", your FAIL_ARCH should then say "if the architecture is not found amongst the four, then fail":

```
FAIL_ARCH="!(arm64|loongarch64|ppc64el|riscv64)"
```